src/ClusterBootstrap/az_tools.py

```diff
@@ -718,14 +741,13 @@ def gen_cluster_config(output_file_name, output_file=True, no_az=False):
     cc["deploydockerETCD"] = False
     cc["platform-scripts"] = "ubuntu"
     cc["basic_auth"] = "%s,admin,1000" % uuid.uuid4().hex[:16]
-    domain_mapping = {"regular":"%s.cloudapp.azure.com" % config["azure_cluster"]["azure_location"], "low": config["domain_name"]}
+    domain_mapping = {"regular":"%s.cloudapp.azure.com" % config["azure_cluster"]["azure_location"], "low": "redmond.corp.microsoft.com"}
```

Comment:
why do we use hard coded "redmond.corp.microsoft.com" ?

Reply:
this is an error happened when I was cherrypicking my changes